src/c++/vdo/tests/RecoveryJournal_t2.c

```diff
@@ -321,6 +336,23 @@ static BlockPattern corruptHeaderPattern[JOURNAL_BLOCKS] = {
   { 0, 0, GOOD_COUNT, USE_NONCE,    FULL_BLOCK, false, normalSectors },
 };
 
+/**
+ * A non-wrapped journal that has been corrupted by the old header
+ * version being written in block 4, along with a hole in block 4 due
```

Comment:
Test comment is technically wrong (sorry), it's a wrapped journal now  (Could just say "A journal")
I'd also dro reference s to "block 4" specfically,  just say "... corrupted by a hole with the old header version. (Being a hole implies the older sequence number.)

Reply:
Fixed.